docs/pages/develop/development-builds/expo-go-to-dev-build.mdx

```diff
@@ -0,0 +1,116 @@
+
+> After generating your `/ios` and/or `/android` directories, make sure you add them to `.gitignore` instead of checking them into Git. This ensures you can always regenerate the code locally or on CI when needed and never have to edit native code manually.
+
+### Option 1: Build on your local machine
+
+To build a native app on your local machine, follow the guide on [setting up your local development environment](https://reactnative.dev/docs/set-up-your-environment) for which ever platform you're looking to build for. This involves setting up and configuring native build tools like Xcode for iOS, and Android Studio for Android.
+
+Once you have everything set up, run the following command to build for iOS:
+
+<Terminal cmd={['$ npx expo run:ios']} />
+
+Or to build the Android app:
+
+<Terminal cmd={['$ npx expo run:android']} />
+
+By default this will build and install the app on a simulator/emulator. If you need to run the build on your phone, plug it into your computer (trust device and allow USB debugging if prompted) and run the above commands with the `--device` flag.
+
+### Option 2: Build on EAS
+
+Building on EAS servers is useful when:
+
+- you can't or don't want to set up your local development environment
+- you want to build an iOS app but don't own a Mac
+- you want to share the development builds with your team
+
+To build on EAS, install the EAS CLI and log in:
+
+<Terminal cmd={['$ npm install -g eas-cli && eas login']} />
+
+To build the development client for Android, run:
+
+<Terminal cmd={['$ eas build --platform android --profile development']} />
+
+To build the development client for iOS (this requires an Apple Developer Account), run:
+
+<Terminal cmd={['$ eas build --platform ios --profile development']} />
+
+See the [next guide](/develop/development-builds/create-a-build) for details on various ways to build your dev client and the limitations.
+
+</Step>
+
+<Step label="2">
+
+## Start the JavaScript bundler
+
+The building locally, `npx expo run:<platform>` will start the bundler automatically. But if you closed the bundler or are working on a dev client you build earlier, (re)start the JavaScript bundler with:
+
+<Terminal cmd={['$ npx expo start']} />
+
+When your project has `expo-dev-client` installed, the bundler will print out **Using development build**, and the QR code it shows will link into the development build you created instead of Expo Go.
+
+</Step>
+
+## Prebuild
+
+**Prebuild** is a concept unique to managed Expo projects. It refers to the process of generating the `/ios` and `/android` directories based on your local config and properties.
+
+### When **not** to run prebuild
+
+All Expo build tools (`eas build` and `npx expo run:<platform>`) will **prebuild** automatically if no existing native folders are found. This means that there is no need to run prebuild manually when running `eas build` or when you're running `npx expo run:<platform>` for the first time.
```

Comment:
To me "when not to" reads like "when you shouldn't" but this is more like "don't need to", but running is still fine. Maybe the wording could be tweaked a bit?

It maybe better to emphasize when to run prebuild first, and add this one as a paragraph at the end since this part isn't that important.